website/templates/log_templates.mako

```diff
@@ -204,15 +204,13 @@ to
 </script>
 
 <script type="text/html" id="project_registered">
-   <!-- ko if: params.is_prereg -->
-        initiated registration of
-        <a class="log-node-title-link overflow" data-bind="text: nodeTitle, attr: {href: projectUrl}"></a>.
-        It was submitted for review to the Preregistration Challenge on
-    <!-- /ko -->
-    <!-- ko ifnot: params.is_prereg -->
         registered
         <a class="log-node-title-link overflow" data-bind="attr: {href: nodeUrl}, text: nodeTitle"></a>
-    <!-- /ko -->
+</script>
+
+<script type="text/html" id="prereg_registration_initiated">
+    submitted for review to the Preregistration Challenge a registration of
```

Comment:
Grammar fix: should be 

"submitted <name> for review to the Preregistration Challenge."